operations/configuration.md

```diff
@@ -21,6 +22,7 @@ Returns configuration of the enterprise and the client.
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
+| `EnterpriseId` | string | optional | Identifies the enterprise to return when multiple are accessible. |
```

Comment:
```suggestion
| `EnterpriseId` | string | optional | Unique identifier of the [Enterprise](#enterprise), required when the access token grants access to multiple enterprises. |
```
This would be my suggestion. BTW, I assume this parameter becomes 'required' in the multi-enterprise case.

Reply:
It is not required because the default value is returned in case the parameter is not specified.